content/en/monitors/notify/variables.md

```diff
@@ -303,6 +202,163 @@ This is the escalation message @dev-team@company.com
 {{% /tab %}}
 {{< /tabs >}}
 
+
+## Notification message enrichment
+
+### Attribute and tag variables
+
+#### Multi-alert variables
```

Comment:
I'm a big fan of Screenshots to illustrate something, we should likely find a way to show the group by in the query and the notification message using its value.